### ISSUE TITLE
MacOSX building does't work

### ISSUE THREAD

**Issue by mecwerks** [fofix/fofix#20]
I tried installing dependencies and compiling from source to help write the mac build instructions but i cannot figure out how to get it to build. I can do testing on mac. My computer is running 10.9

**Comment by fculpo** [comment]
I own a Mac, and managed to build from source (dependencies installed via brew)
i should be able to do some more tests soon

**Comment by mdsitton** [comment]
Yeah the game will run from sources, and most everything works. The only issue that I know of right now is that building a packaged version with py2app is still broken (otherwise I would have closed this already)

**Comment by marcwebbie** [comment]
What are the brew packages you installed @fculpo ? Do you have a list somewhere?
